src/Components/Form/index.jsx

```diff
@@ -4,15 +4,15 @@ import './Form.scss';
 function Form(props) {
   const [url, setUrl] = useState('');
   const [method, setMethod] = useState('GET');
-  const [postData, setPostData] = useState('');
+  const [data, setData] = useState('');
 
 
   const handleSubmit = (e) => {
     e.preventDefault();
     const formData = {
       method: method,
       url: url,
-      data: postData,
+      // data: postData,
```

Comment:
Our data is currently a string since it was input via text area; we want to turn it into an object so axios knows how to properly send it as JSON. 
``` JavaScript
const formData = {
  method: method,
  url: url,
  data: JSON.parse(data),
};